src/Html.php

```diff
@@ -1122,9 +1122,10 @@ public static function includeHeader(
         self::header_nocache();
 
         $theme = $_SESSION['glpipalette'] ?? 'auror';
+        $lang = $_SESSION['glpilanguage'] ?? $CFG_GLPI['language'];
```

Comment:
This method will adapt the language to the client configuration.

```suggestion
        $lang = $_SESSION['glpilanguage'] ?? Session::getPreferredLanguage();
```